src/containerapp/azext_containerapp/_utils.py

```diff
@@ -428,6 +428,48 @@ def get_linker_client(cmd):
     return linker_client
 
 
+def linker_create_or_update(linker_client, r, parameters, linker_name):
```

Comment:
We should check the result in `linker_create_or_update `and `delete_managed_binding` to make sure the execution result is successful, right?

Reply:
you're right. it's a good sanity check. will add!!